modules/logging-enabling-loki-alerts.adoc

```diff
@@ -38,26 +38,26 @@ The `AlertingRule` CR contains a set of specifications and webhook validation de
   kind: AlertingRule
   metadata:
     name: loki-operator-alerts
-    namespace: openshift-operators-redhat <1>
-    labels: <2>
-      openshift.io/<label_name>: "true"
+    namespace: openshift-operators-redhat #<1>
+    labels: #<2>
+      openshift.io/cluster-monitoring: "true"
   spec:
-    tenantID: "infrastructure" <3>
+    tenantID: infrastructure #<3>
     groups:
       - name: LokiOperatorHighReconciliationError
         rules:
           - alert: HighPercentageError
-            expr: | <4>
+            expr: | #<4>
               sum(rate({kubernetes_namespace_name="openshift-operators-redhat", kubernetes_pod_name=~"loki-operator-controller-manager.*"} |= "error" [1m])) by (job)
                 /
               sum(rate({kubernetes_namespace_name="openshift-operators-redhat", kubernetes_pod_name=~"loki-operator-controller-manager.*"}[1m])) by (job)
                 > 0.01
             for: 10s
             labels:
-              severity: critical <5>
+              severity: critical #<5>
             annotations:
-              summary: High Loki Operator Reconciliation Errors <6>
-              description: High Loki Operator Reconciliation Errors <7>
+              summary: High Loki Operator Reconciliation Errors #<6>
+              description: High Loki Operator Reconciliation Errors #<7>
 ----
 <1> The namespace where this `AlertingRule` CR is created must have a label matching the LokiStack `spec.rules.namespaceSelector` definition.
 <2> The `labels` block must match the LokiStack `spec.rules.selector` definition.
```

Comment:
The line below this one seems to have another issue with additional `\` being displayed.